EmployeeList/Views/EmployeeListRow.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  EmployeeListRow.swift
+//  EmployeeList
+//
+//  Created by Wang Siman on 1/29/21.
+//  Copyright © 2021 None. All rights reserved.
+//
+
+import SwiftUI
+import UIKit
+
+struct EmployeeListRow: View {
+    @State var employee: Employee
+
+    var body: some View {
+        HStack {
+            CustomImageView(customImageViewModel: .init(imageURL: employee.photoUrlSmall!))
+            Text(employee.fullName)
+            Spacer()
+            Text(employee.team)
+                .foregroundColor(.gray)
+        }
+        .padding()
+    }
+}
+
+struct EmployeeListRow_Previews: PreviewProvider {
+    static var previews: some View {
+        EmployeeListRow(employee:
+            Employee(
```

Comment:
I would create a static var on Employee with this info, and then use it wherever you need a placeholder like this.

Reply:
This is smart!